src/validations/profileSchema.js

```diff
@@ -0,0 +1,95 @@
+import { check } from 'express-validator/check';
+import makeUpperCase from '../helpers/makeUpperCase';
+
+const profileSchemas = {
+  profileUpdateSchema: [
+    check('userName')
+      .optional()
+      .trim()
+      .not()
+      .isEmpty({ ignore_whitespace: true })
+      .withMessage('userName cannot be blank')
+      .isLength({ min: 2, max: 20 })
+      .withMessage('username must be at least 2 characters, and maximum 20')
+      .not()
+      .matches(/^[A-Za-z]+[-]{1}[A-Za-z]+([-]{1}[A-Za-z]+)+$/, 'g')
+      .withMessage('invalid input for username')
+      .not()
+      .matches(/^[A-Za-z]+[']+[A-Za-z]+[']+[A-Za-z]+$/, 'g')
```

Comment:
Why do we have 3 different regex validations? I think it's better to use just 1 and make it as inclusive and exclusive as is needed.

Reply:
Done.